packages/bap-abi/bap-abi.2.5.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "bap-abi"
+version: "2.5.0"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["./configure" "--prefix=%{prefix}%" "--enable-abi"]
+  [make]
+]
+install: [[make "install"]]
+remove: [["ocamlfind" "remove" "bap-plugin-abi"]
+         ["ocamlfind" "remove" "bap-abi"]
+         ["bapbundle" "remove" "abi.plugin"]
+         ]
+depends: [
+  "ocaml" {>= "4.08.0" }
+  "core_kernel" {>= "v0.14" & < "v0.16"}
+  "ppx_bap"
+  "bap-std" {= "2.5.0"}
```

Comment:
As usual: it would probably be easier for you to use `version` instead of bumping all of the constraints by hand everytime:
```suggestion
  "bap-std" {= version}
```